trpc/routers/buildings.ts

```diff
@@ -0,0 +1,53 @@
+import { procedure, router } from '../init';
+import { prisma } from '@/prisma/prisma';
+import { z } from 'zod';
+import { buildingDOtoDTO, buildingEntryDOtoDTO } from '@/types/Building';
+
+export const buildingsRouter = router({
+  getBuildings: procedure.query(async () => {
+    const buildings = await prisma.building.findMany({
+      include: {
+        building_entries: true,
+      },
+    });
+    return buildings.map(buildingDOtoDTO);
+  }),
+
+  getBuildingById: procedure
```

Comment:
Shouldn't this route be incorporated into `getBuildings`, with an optional ID? AKA:
- if we don't pass the id, we get all the buildings
- if we pass the id, we return only the one